src/openforms/emails/templates/emails/cosign.html

```diff
@@ -0,0 +1,25 @@
+{% load static i18n %}
+
+<p>
+    {% blocktranslate %}
+        This is a request to co-sign form "{{ form_name }}".<br />
+    {% endblocktranslate %}
+    
+    {% if show_form_link %}
+        {% blocktranslate %}
+            Please click the link below to co-sign:<br />
+            <a class="utrecht-link utrecht-link--openforms" href="{{ form_url }}">Co-sign</a><br />
```

Comment:
Maybe the text of the message could be the same as for when the link is not enabled, except with an extra:
```
You can go to the form page also by clicking on <a class="utrecht-link utrecht-link--openforms" href="{{ form_url }}">this limk</a>
```
or something... :stuck_out_tongue: Maybe @LaurensBurger has a better suggestion